### PR TITLE
fix: bug in vgg16_fp8_ptq example

### LINKED ISSUE #2932: Should "model" be "trt_model" here?
Might be a copy paste error:

https://github.com/pytorch/TensorRT/blob/52ba6f1e3ff5b905e15a12b500af2d4abf847e21/examples/dynamo/vgg16_fp8_ptq.py#L239

above has this code:
```python
        trt_model = torchtrt.dynamo.compile(
            exp_program,
            inputs=[input_tensor],
            enabled_precisions={torch.float8_e4m3fn},
            min_block_size=1,
            debug=False,
        )
```

and then a note saying

```
        # Inference compiled Torch-TensorRT model over the testing dataset
```

but then it looks like inference is done with the original model?

```python
            out = model(data)
```

### PR DESCRIPTION
**Pull request by zewenli98**
# Description

Fix a bug in `vgg16_fp8_ptq.py` example.

Fixes #2932 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: examples/dynamo/vgg16_fp8_ptq.py [hunk 1]
```diff
@@ -235,10 +243,10 @@ def calibrate_loop(model):
         loss = 0.0
         class_probs = []
         class_preds = []
-        model.eval()
+        trt_model.eval()
```
Comment (peri044) [inline examples/dynamo/vgg16_fp8_ptq.py:246]: You can remove this line

**Review by peri044 — APPROVED** [review]
minor change. LGTM
